### ISSUE TITLE
word repeat rule should also cover sentence before and after the current one.

### ISSUE THREAD

**Issue by kostyfisik** [languagetool-org/languagetool#597]
Example - sequence usage of "был получен":
Такие значения показателя преломления обычно недостижимы в оптическом диапазоне
частот, но могут **быть получены** для микроволнового диапазона. **Были получены** графики в зависимости от толщины покрытия.

**Comment by yakovru** [comment]
Word "быть", "были" is very common in Russian. This word **must** excluded from same rules.
Russian verb "быть" = English verb "to be". This verb uses for make future simple tense in Russian: "буду играть".

**Comment by kostyfisik** [comment]
@yakovru You are right! http://dict.ruslang.ru/freq.php?act=show&dic=freq_v Still "получен" is not in the top of this list.
